src/components/icons/UrlIcon.tsx

```diff
@@ -11,8 +11,7 @@ const UrlIcon = () => {
       viewBox='0 0 30 30'
       style={{
         marginLeft: '2px',
-        marginTop: '-0.056em',
-        padding: '0.167em 0.167em 0.167em 0.056em',
```

Comment:
The padding increases the clickable area to make it easier for the user to tap, so we should keep it.